django_enumfield/tests/test_enum.py

```diff
@@ -32,16 +33,19 @@ def test_enum_field_save(self):
 
         self.assertRaises(InvalidStatusOperationError, setattr, lamp, 'state', 99)
 
-        # Test model with EnumField WITH _transitions
+        # Test model with EnumField WITH __transitions__
         person = Person.objects.create()
         pk = person.pk
         self.assertEqual(person.status, PersonStatus.ALIVE)
         person.status = PersonStatus.DEAD
         person.save()
+        self.assertTrue(isinstance(person.status, PersonStatus))
         self.assertEqual(person.status, PersonStatus.DEAD)
 
         person = Person.objects.get(pk=pk)
         self.assertEqual(person.status, PersonStatus.DEAD)
+        # TODO: find a way to get only one type
+        self.assertTrue(isinstance(person.status, (PersonStatus, int)))
```

Comment:
It doesn't help unfortunately, however it may be a good idea to subclass `IntEnum` anyway since we currently work with integers only. In future, we will add support for string enums and it may be time to start making some preparations for it as well so we have an easy structure to extend.